features/sriov/deploy.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+set -uo pipefail
+BASEDIR="$(dirname "$0")"
+
+# shellcheck disable=SC1091,SC1090
+. "${BASEDIR}/../../lib/functions.sh"
+
+# shellcheck disable=SC1091,SC1090
+. "${BASEDIR}/myvars"
+
+apply_manifest(){
+  envsubst < ./*-sriov-"${1}".yaml | oc apply -f > /dev/null || die "Error creating ${1}"
+}
+
+deploy(){
+  for object in namespace operatorgroup subscription; do
+    apply_manifest ${object}
+  done
+}
+
+# This is needed for multus to create a dhcp-daemon daemonset in order for ipam/dhcp to work
+add_dummy_dhcp(){
+  oc patch networks.operator.openshift.io cluster --type='merge' \
```

Comment:
if it was previously run, it will die instead of following on

Reply:
I will add a check that to verify the `dummy-dhcp-network` doesn't exists before trying to create it. Thanks.